src/components/rpc_base/include/rpc_base/rpc_base_json_inl.h

```diff
@@ -164,7 +164,7 @@ Json::Value Integer<T, minval, maxval>::ToJsonValue() const {
 
 template <int64_t minnum, int64_t maxnum, int64_t minden, int64_t maxden>
 Float<minnum, maxnum, minden, maxden>::Float(const Json::Value* value)
-    : PrimitiveType(InitHelper(value, &Json::Value::isDouble)), value_() {
+    : PrimitiveType(InitHelper(value, &Json::Value::isNumeric)), value_() {
```

Comment:
Why was this changed?

Reply:
Json::Value::isDouble("4")  will return false. .
But 4 is correct double.